lib/puppet/functions/hiera_vault.rb

```diff
@@ -177,31 +174,28 @@ def lookup_key(key, options, context)
     end
   end
 
-
   def vault_get(key, options, context)
-
-    if ! ['string','json',nil].include?(options['default_field_parse'])
+    if !['string', 'json', nil].include?(options['default_field_parse'])
       raise ArgumentError, "[hiera-vault] invalid value for default_field_parse: '#{options['default_field_parse']}', should be one of 'string','json'"
     end
 
-    if ! ['ignore','only',nil].include?(options['default_field_behavior'])
+    if !['ignore', 'only', nil].include?(options['default_field_behavior'])
       raise ArgumentError, "[hiera-vault] invalid value for default_field_behavior: '#{options['default_field_behavior']}', should be one of 'ignore','only'"
     end
 
-    if (! options['cache_for'].nil?) &&  (! options['cache_for'].is_a? Numeric)
+    if (!options['cache_for'].nil?) && (!options['cache_for'].is_a? Numeric)
```

Comment:
Isn't this easier to read?
```suggestion
    if !options['cache_for'].nil? && !options['cache_for'].is_a?(Numeric)
```

Taking that a step further (and closer aligning to the shown error message):
```suggestion
    if !(options['cache_for'].nil? ||options['cache_for'].is_a?(Numeric))
```

Though this file doesn't really use `unless`, it could then be written as:
```suggestion
    unless options['cache_for'].nil? ||options['cache_for'].is_a?(Numeric)
```